presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSplitManager.java

```diff
@@ -93,7 +119,7 @@ public ConnectorSplitSource getSplits(
                     .metricsReporter(new RuntimeStatsMetricsReporter(session.getRuntimeStats()))
                     .fromSnapshotExclusive(fromSnapshot)
                     .toSnapshot(toSnapshot);
-            return new ChangelogSplitSource(session, typeManager, icebergTable, scan);
+            return new ChangelogSplitSource(session, typeManager, icebergTable, scan, icebergTable.snapshot(toSnapshot).sequenceNumber());
```

Comment:
```suggestion
            return new ChangelogSplitSource(session, typeManager, icebergTable, scan);
```

nit: the sequence number seems unnecessary here